crates/aztec_backend/src/serialiser/mod.rs

```diff
@@ -372,6 +372,7 @@ pub fn serialise_circuit(circuit: &Circuit) -> ConstraintSystem {
 
                         fixed_base_scalar_mul_constraints.push(fixed_base_scalar_mul);
                     }
+                    OPCODE::ToBits => unreachable!("to_bits is not supported in Barretenberg"),
```

Comment:
I guess support could be added in Barretenberg, but ultimately it will depend on what  would be the benefit from having it done in Barretenberg vs Noir.